bin/main.js

```diff
@@ -15,9 +16,15 @@ process.argv.slice(2).forEach(function (val, index, array) {
     if (val === 'static') {
         staticMode = true;
     }
+
+    if (val.includes('once')) {
```

Comment:
@heydarli that is bit confusing, let's have consistent arguments without - or --

`candygen static` 
`candygen once`
etc.